### PR TITLE
Fix build file name

### PR DESCRIPTION
**Pull request by suneeta-mall**
Fixes the case of duplicate file/folder "build". 

Also mentioned in https://github.com/pachyderm/python-pachyderm/issues/20#issuecomment-394138052

### CODE REVIEW

**Comment by suneeta-mall** [comment]
Fixes issues mentioned in https://github.com/pachyderm/python-pachyderm/issues/24 and https://github.com/pachyderm/python-pachyderm/issues/20.

**Comment by suneeta-mall** [comment]
@jdoliner Are you happy with this fix ^^. 

Also, I want to get this in Conda-forge to manage my installations. Would you mind I raise a PR againts conda-forge (meta.yaml) pointing to your pypi artefacts ? or you prefer to manage that yourself?

**Comment by jdoliner** [comment]
@suneeta-mall very happy with the fix, thank you so much. Merging.

We'd love to be on conda-forge, and would appreciate it if you'd open up the PR for us on conda-forge. Also, if you wouldn't mind, @ me on the PR so I can +1 and monitor it in case there's any Pachyderm specific questions that come up.

Thanks so much for helping us smooth up the Python client, we don't have a ton of internal python knowledge so we're relying on the community to help us figure out where it needs to go.

**Comment by jdoliner** [comment]
Also, should #24 and #20 be closed after merging this?

**Comment by suneeta-mall** [comment]
@jdoliner  sure. I will do the conda-forge thingie. 

As for #24 and #20 yeah they can be closed. ta

File: Makefile [hunk 1]
```diff
@@ -59,7 +59,7 @@ sync:
 
 release:
 	# Bump the version
-	expr $$(cat BUILD) + 1 > BUILD
+	expr $$(cat BUILD) + 1 > BUILD_NUMBER
```
Comment (itssimon) [inline Makefile]: I believe this line should read
`expr $$(cat BUILD_NUMBER) + 1 > BUILD_NUMBER`

Comment (suneeta-mall) [reply Makefile]: oh bummber! Yes .. I will fix it .. ta

Comment (suneeta-mall) [reply Makefile]: Fixed in https://github.com/pachyderm/python-pachyderm/pull/27/files Thanks @itssimon

**Pull request merged**